core/src/main/java/org/mskcc/cbio/portal/scripts/ImportFusionData.java

```diff
@@ -138,6 +138,7 @@ public void importData() throws IOException, DaoException {
                     // TODO we may need get mutation type from the file
                     // instead of defining a constant
                     mutation.setMutationType(FUSION);
+                    mutation.setMutationStatus(record.getFusionStatus().toUpperCase());
```

Comment:
a little confusing .. (set mutation status to fusion status) .. but the real issue is presenting fusions under datatype mutation. We are lucky that we can reuse the "germline" flag of mutations for this datatype.

Reply:
This looks like an effective solution to support this type of data.

I'm wondering how this will look in the OncoPrint. We recently added Germline visualization in the OncoPrint, but obviously, we haven't tested this with a Fusion event. Could you check if it works as expected?

Example of germline mutation in oncoprint:
http://www.cbioportal.org/index.do?session_id=5af5a304498eb8b3d56615b6